src/store/user/index.js

```diff
@@ -236,9 +248,15 @@ const actions = {
     },
 
     async getUserProfile({ commit, dispatch, state }) {
+        let userType = localStorage.getItem('UserType');
+        if (userType !== "undefined" && userType !== "null") {
+            commit('set-user-type', userType);
```

Comment:
which the userType is already present in localStorage, why are  you setting it again.